app_runner.rb

```diff
@@ -56,4 +70,73 @@ def handle_list_rentals_for_person
     person_id = gets.chomp.to_i
     @app.list_rentals_for_person(person_id)
   end
+
+  def read_data_from_file(filename)
+    data = []
+    begin
+      file = File.open(filename, 'r')
+      data = JSON.parse(file.read)
+    rescue StandardError
+      file = File.open(filename, 'w')
+      file.write('[]')
+      file.close
+    end
+    data
+  end
+
+  def store_data(data, filename)
+    object_data = []
+    data.each do |o|
+      object_data << if o.is_a?(Rental)
+                       {
+                         book_title: o.book.title,
+                         person_id: o.person.id,
+                         date: o.date.strftime('%Y-%m-%d')
+                       }
+                     else
```

Comment:
- [ ] When I exit the app, it is expected to save the data I have input, but the strftime object is throwing errors and it gave suggestions, I will recommend you double-check it to avoid the error 
![image](https://github.com/theresetuyi/OOP-school-library/assets/86200356/2a9eb6d0-0702-4183-af49-070c2d77f2ae)

This causes issues when I want to relaunch the app.